apps/studio/components/layouts/TableEditorLayout/EntityListItem.tsx

```diff
@@ -29,6 +29,8 @@ import {
 import { useProjectContext } from '../ProjectLayout/ProjectContext'
 import { useProjectLintsQuery } from 'data/lint/lint-query'
 import { getEntityLintDetails } from 'components/interfaces/TableGridEditor/TableEntity.utils'
+import useTableDefinition from '../../../hooks/misc/useTableDefinition'
```

Comment:
should be able to import these like this?

https://github.com/supabase/supabase/blob/master/apps/studio/components/interfaces/TableGridEditor/GridHeaderActions.tsx#L22

Reply:
I'll change the import as it is in the given example.
Got mine from somewhere else where it's also done the way I did it, but how it's done in the example is much cleaner.